_includes/templates/download.html

```diff
@@ -88,10 +88,9 @@ <h2>{{ page.latestversion }} {{CURRENT_RELEASE}} <a type="application/rss+xml" h
       <div>
         <img src="/assets/images/os/med_linux.png" alt="linux">
         <span>
-          <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.ppc64le }}.tar.gz" class="dl">PPC64 Linux</a>
-            <span><a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.ppc64 }}.tar.gz" class="dl" id="lin64ppc">64 bit</a>
-            -
-            <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.ppc64le }}.tar.gz" class="dl" id="lin64ppcle">64 bit LE</a></span>
+          <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.ppc64 }}.tar.gz" class="dl">PPC64 Linux</a>
+            <span><a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}-{{ site.data.binaries.ppc64 }}.tar.gz" class="dl" id="lin64ppc">64 bit</a></span>
+            <!-- PPC64LE Disabled for now due to guix build issue, removed from template as the binary availability check in CI doesn't ignore comments -->
```

Comment:
here's how it displays for me
<img width="386" alt="image" src="https://github.com/user-attachments/assets/0af26181-2a9c-4958-bd16-a19d7cf9a149">